app/controllers/admin/volunteers.py

```diff
@@ -97,34 +96,37 @@ def trackVolunteersPage(eventID):
 
 
 
-@admin_bp.route('/event/<eventID>/dietary_restrictions', methods=['GET'])
-def dietaryRestrictionsPage(eventID):
+@admin_bp.route('/event/<eventID>/volunteer_information', methods=['GET'])
+def volunteerInformationPage(eventID):
     try:
         event = Event.get_by_id(eventID)
     except DoesNotExist as e:
         print(f"No event found for {eventID}", e)
         abort(404)
 
-
     if not (g.current_user.isCeltsAdmin or (g.current_user.isCeltsStudentStaff and g.current_user.isProgramManagerForEvent(event))):
         abort(403)
 
-    eventRsvpData = list(EventRsvp.select().where(EventRsvp.event==event).order_by(EventRsvp.rsvpTime))
+    eventRsvpData = list(EventRsvp.select().where(EventRsvp.event==event))
     eventParticipantData = list(EventParticipant.select().where(EventParticipant.event==event))
     participantsAndRsvp = (eventParticipantData + eventRsvpData)
-
-    #get unique list of users for each category waitlist/notwaitlist
-    volunteerUser = list(set([obj.user for obj in participantsAndRsvp if not obj.rsvpWaitlist and obj.user.dietRestriction]))
-    waitlistUser = list(set([obj.user for obj in participantsAndRsvp if obj.rsvpWaitlist and obj.user.dietRestriction]))
-
+    eventParticipantUsers = [obj.user for obj in eventParticipantData]
+    eventNonAttendedData = [obj.user for obj in eventRsvpData if obj.user not in eventParticipantUsers]
+    
+    #get unique list of users for each category waitlist/notwaitlist,rsvped/attended
+    volunteerUser = list(set([obj.user for obj in eventParticipantData if not obj.rsvpWaitlist]))
```

Comment:
You have essentially replaced this with `attendedUser`. You also never use this anywhere else so if it is not needed remove it.

Reply:
You are correct, we've removed volunteerUser.